src/components/markdown/markdown.tsx

```diff
@@ -119,9 +119,30 @@ export const Markdown = forwardRef<HTMLDivElement, MarkdownProps>(
       const componentMapWithPassedThroughProps = Object.entries(
         componentMap
       ).reduce((newMap, [key, Component]) => {
-        newMap[key] = (props: any) => (
-          <Component {...props} {...(componentProps || {})} />
-        );
+        newMap[key] = (props: any) => {
+          // Replace \n with <br /> for paragraphs
+          const children =
+            key === 'p'
+              ? props?.children?.map((child: any, i: number) => {
+                  if (child === '\n') return <br key={i} />;
```

Comment:
Windows newlines?

Reply:
Yes, carriage returns weren't being rendered. The text string accurately reflects the line break inside the inspector, but the slide itself renders the text as inline.

Reply:
I thought the Windows sequence was `\r\n`, which we might want to test for as well. It has the carriage return then new feed flags.

Reply:
Oh, I see. I didn't realize Windows had a different sequence.

Reply:
Yep! And let's add test cases for win and mixed win+mac/lin

Reply:
I just put in a fix.